dash-renderer/jest.init.js

```diff
@@ -0,0 +1 @@
+import '@babel/polyfill';
```

Comment:
jest now needs the polyfills..

Reply:
seems fine, but I'm surprised jest is running in an environment that needs polyfills.

Reply:
jest doesn't, but jest needs to polyfills to handle our code!